packages/office-ui-fabric-react/src/components/Callout/__snapshots__/Callout.test.tsx.snap

```diff
@@ -2,30 +2,22 @@
 
 exports[`Callout renders Callout correctly 1`] = `
 <div
-  className="ms-Callout-container"
+  className={undefined}
```

Comment:
some of the global class names are being dropped off here. Is this being tested correctly? You shouldn't see a big change in the snapshot tests.

Reply:
What do you mean by tested correctly, unit tests?

Reply:
I thing this happened because I used CalloutContentBase in the tests instead of CalloutContent. After changing that, I get the classes back.